content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Quick Start"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* This step-by-step guide will walk you through creating your first SeaChat agent from scratch. In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
```

Comment:
@itIsJim We want to show how to create AI agent in 10 minutes **with an actual example**. We want to use the parking example as a template to **show** people a real use case. The example files for the parking bot from Kim should be a part of this tutorial.

Just like the tutorial we all liked: https://www.youtube.com/watch?v=Edj6ijQI1hY. Let's demo an actual use case, not just listing the steps and buttons on this tutorial
<img width="1266" alt="Screenshot 2024-03-11 at 17 51 48" src="https://github.com/seasalt-ai/wiki/assets/20276372/086a7736-4160-4cae-a134-ce7e21581772">